src/replacer.coffee

```diff
@@ -55,7 +58,8 @@ class Replacer
   _replaceArray: (orig) ->
     arr = (@replace(item) for item in orig)
     # Convert the nextPage methods for paged results
-    for key, value of orig
+    for key in Object.keys(orig)
+      value = orig[key]
       @_replaceKeyValue(arr, key, value) if typeof key is 'string'
```

Comment:
I would still say that this line make no sense. Key is always a string. You resolve the issue of #55 because you only iterate over it's own properties, but it still call replaceKeyValue for ALL properties.